full/src/test/java/apoc/redis/RedisTest.java

```diff
@@ -0,0 +1,375 @@
+                map("uri", URI, "value", numberValue, "mapKey", key, "config", config),
+                r -> assertEquals(true, r.get("value")));
+
+        TestUtil.testCall(db, "CALL apoc.redis.hget($uri, $mapKey, $value, $config)",
+                map("uri", URI, "value", numberValue, "mapKey", key, "config", config),
+                r -> assertEquals("1", fromCodec(r.get("value"))));
+
+        TestUtil.testCall(db, "CALL apoc.redis.hincrby($uri, $mapKey, $value, 3, $config)",
+                map("uri", URI, "mapKey", key, "value", numberValue, "config", config),
+                r -> assertEquals(4L, r.get("value")));
+
+        TestUtil.testCall(db, "CALL apoc.redis.hgetall($uri, $mapKey, $config)",
+                map("uri", URI, "mapKey", key, "config", config),
+                r -> {
+                    Map<String, String> actualMap = ((Map<String, Object>) r.get("value")).entrySet()
+                            .stream()
+                            .collect(Collectors.toMap(Map.Entry::getKey, i -> fromCodec(i.getValue())));
+                    assertEquals(map("number", "4"), actualMap);
+                });
+    }
+
+    @Test
+    public void testEvalCommand() {
+        Object keyEval = getByCodec("testEval");
+        TestUtil.testCall(db, "CALL apoc.redis.setGet($uri, $keyEval, $valueEval, $config)",
+                map("uri", URI, "keyEval", keyEval, "valueEval", getByCodec("valueEval"), "config", map("codec", codec)), 
+                r -> assertNull(r.get("value")));
+        
+        TestUtil.testCall(db, "CALL apoc.redis.eval($uri, 'return redis.call(\"get\", KEYS[1])', 'VALUE', [$keyEval], [$keyName], $config)", 
+                map("uri", URI, "keyEval", keyEval, "keyName", getByCodec("key:name"), "config", map("codec", codec)),
+                r -> assertEquals("valueEval", fromCodec(r.get("value"))));
+    }
+
+    @Test
+    public void testKeysCommand() {
+        Map<String, Object> config = map("codec", this.codec);
+        Object from = getByCodec("from");
+        Object to = getByCodec("to");
+        TestUtil.testCall(db, "CALL apoc.redis.setGet($uri, $from, $one, $config)",
+                map("uri", URI, "from", from, "one", getByCodec("one"), "config", config),
+                r -> assertNull(r.get("value")));
+
+        TestUtil.testCall(db, "CALL apoc.redis.copy($uri, $from, $to, $config)",
+                map("uri", URI, "from", from, "to", to, "config", config),
+                r -> assertEquals(true, r.get("value")));
+        
+        TestUtil.testCall(db, "CALL apoc.redis.get($uri, $to, $config)",
+                map("uri", URI, "to", to, "config", config),
+                r -> assertEquals("one", fromCodec(r.get("value"))));
+        
+        TestUtil.testCall(db, "CALL apoc.redis.exists($uri, [$to], $config)",
+                map("uri", URI, "to", to, "config", config),
+                r -> assertEquals(1L, r.get("value")));
```

Comment:
since `apoc.redis.copy` returns a true boolean, why does `apoc.redis.exists` return 1 here?

Reply:
Also in this case, the java client works differently, because `copy` returns true or false instead of 1 or 0  https://redis.io/commands/COPY command.
I could change the `@Description`, but maybe in this case it is worth changing the return value to make it consistent with the `COPY` command. Wdyt?

Reply:
Let's leave it consistent with the Java client. Otherwise, we should change them all to match Redis client.